Source/Core/Core/AchievementManager.cpp

```diff
@@ -426,6 +691,8 @@ AchievementManager::GetUnlockStatus(AchievementId achievement_id) const
 void AchievementManager::GetAchievementProgress(AchievementId achievement_id, u32* value,
```

Comment:
If you're gonna add an early-out condition here then this should probably return a boolean to indicate success/failure. Maybe also check the return value of `rc_runtime_get_achievement_measured()` while you're here.

Reply:
Gonna spin this into a separate PR. You're right, but I only added this here because I suddenly noticed it and it has nothing to do with the badges.

Reply:
Yes, please do.

Reply:
https://github.com/dolphin-emu/dolphin/pull/12212